Makefile

```diff
@@ -1,4 +1,7 @@
-check: fmt build test docs
+# Do test-loop last, since the tests for that target require sudo.
+# Using sudo changes permissions on various directories. It is less trouble
+# not to have to fix up permissions after every sudo'd test.
+check: fmt build test docs test-loop
```

Comment:
It's there because I'm more interested in the result of test target than in the result of docs target. I've moved it.